server/channels/store/sqlstore/shared_channel_store.go

```diff
@@ -645,9 +698,14 @@ func (s SqlSharedChannelStore) GetUsersForSync(filter model.GetUsersForSyncFilte
 	}
 
 	query := s.getQueryBuilder().
-		Select("u.*").
+		Select("u.Id", "u.CreateAt", "u.UpdateAt", "u.DeleteAt", "u.Username", "u.Email", "u.EmailVerified",
+			"u.Nickname", "u.FirstName", "u.LastName", "u.Position", "u.Roles", "u.AllowMarketing", "u.Props",
+			"u.NotifyProps", "u.LastPasswordUpdate", "u.LastPictureUpdate", "u.Locale", "u.Timezone",
+			"b.UserId IS NOT NULL AS IsBot", "COALESCE(b.Description, '') AS BotDescription",
+			"COALESCE(b.LastIconUpdate, 0) AS BotLastIconUpdate", "u.RemoteId").
 		Distinct().
 		From("Users AS u").
+		LeftJoin("Bots b ON ( b.UserId = u.Id )").
```

Comment:
This seems like a change in behavior? Wondering if it's possible to update the unit test to test the new column?

Reply:
Agreed it is a change, and I've reverted the join to avoid unnecessary change.  Shared channels doesn't use the fields.